common/data_refinery_common/migrations/0013_organism_experiments.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.5 on 2019-02-06 22:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_refinery_common', '0012_experiment_platform_accession_codes'),
```

Comment:
this conflicts with https://github.com/AlexsLemonade/refinebio/pull/1006

Reply:
I think #1006 is ready to go out, I'll resolve this once that gets merged.